Blockzilla/BrowserViewController.swift

```diff
@@ -244,6 +246,13 @@ class BrowserViewController: UIViewController {
             homeView.removeFromSuperview()
         }
         self.homeView = homeView
+        
+        if shouldShowTrackerStatsShareButton() {
```

Comment:
This is run after a user taps "Erase" so there is a chance they would not see it on launch but then see it after they erased a browsing session. Is that ok?

Reply:
A _chance_ they wouldn't see it on launch? And then they would see it every time they 'Erase'?

Reply:
Ahh i miss understood! I thought the 10% rule was just to not show it all the time and annoy users. I will change it to bucket users into groups.